### PR TITLE
update IN translation

### PR DESCRIPTION
**Pull request by ratmgr**

### CODE REVIEW

File: in/index.md [hunk 1]
```diff
@@ -6,29 +6,29 @@ title: Daftar Isi Bantuan
 
 <div class="panel panel-info">
   <div class="panel-heading">
-    <strong>You can't find a solution to your problem? You still have questions?</strong>
```
Comment (koppor) [inline in/index.md]: This is currently not translatable. It is too hard for us to adapt https://github.com/JabRef/help.jabref.org/blob/gh-pages/_scripts/automate.py accordingly.

The templates are residing in https://github.com/JabRef/help.jabref.org/tree/gh-pages/_includes

Can you please translate https://github.com/JabRef/help.jabref.org/blob/gh-pages/_includes/link-to-toc-in.html and https://github.com/JabRef/help.jabref.org/blob/gh-pages/_includes/link-to-main-in.html?

File: in/index.md [hunk 2]
```diff
@@ -97,21 +97,21 @@ title: Daftar Isi Bantuan
 - [XMP metadata support in JabRef](/in/XMP)
 
 
-## Fetching entries from the web
```
Comment (koppor) [inline in/index.md]: Translation does not work like that here.

Please use https://github.com/JabRef/help.jabref.org/blob/gh-pages/in/localization_in.json. Replace `""` by `"your-translation"`.

File: in/index.md [hunk 3]
```diff
@@ -6,29 +6,29 @@ title: Daftar Isi Bantuan
...
-- [Sharing](/in/FAQsharing)
+- [Lainnya](/in/FAQother)
+- [Berbagi](/in/FAQsharing)
 
 
 ## Umum
 - [Backup](/in/Backup)
 - [Best Practices](/in/BestPractices)
 - [Command line use and options](/in/CommandLine)
-- [Installation](/in/Installation)
```
Comment (koppor) [inline in/index.md]: Translation does not work like that in this tool.

Please go into https://github.com/JabRef/help.jabref.org/tree/gh-pages/in and place translated files for https://github.com/JabRef/help.jabref.org/tree/gh-pages/en

For instance, for "Installation", translate https://github.com/JabRef/help.jabref.org/blob/gh-pages/en/Installation.md into https://github.com/JabRef/help.jabref.org/blob/gh-pages/in/Installation.md

**Comment by ratmgr** [comment]
thank you for respond my pull request,
should i change it ?

File: in/index.md (hunk 1 above)
Comment (ratmgr) [reply in/index.md]: I'll try to translate it

**Comment by koppor** [comment]
Admin note: Follow-up at https://github.com/JabRef/help.jabref.org/pull/242

@ratmgr: There is no need to create a new pull request. In the future, just add commits to the current branch. You can also [amend your commit](https://www.atlassian.com/git/tutorials/rewriting-history).

See https://lostechies.com/joshuaflanagan/2010/09/03/use-gitk-to-understand-git/ for a good introduction into git.

**Pull request closed without merging**